api/coreservice.go

```diff
@@ -1376,7 +1385,28 @@ func (core *coreService) getAction(actHash hash.Hash256, checkPending bool) (*io
 	if err != nil {
 		return nil, err
 	}
-	return core.pendingAction(selp)
+	return core.actionToApiProto(selp)
+}
+
+func (core *coreService) checkContract(ctx context.Context, to *common.Address) (bool, bool, bool, error) {
```

Comment:
duplicated with https://github.com/iotexproject/iotex-core/blob/master/state/factory/workingset.go#L250, can add a TODO to refactor it later